contracts/libs/Owned.sol

```diff
@@ -13,5 +13,9 @@ contract Owned {
 
   function isOwner(address addr) public returns(bool) { return addr == owner; }
```

Comment:
@oed As per the security review, we should consider removing this function. It's never used, and we can just move the logic to the onlyOwner modifier.